### DISCUSSION TITLE
Adaptive mesh leveling (AML 2.0)

### DISCUSSION THREAD

**Discussion by 3D-enigma007** [mriscoc/Ender3V2S1#1264] [Q&A]
Good morning, I tried to install AML 2.0 in Cura 5.5 but I don't see it in the script list, if I install the AML 1.0 version I see it in the list and it works, today I updated Cura to 5.6 but I don't find it when I open the script list post processing.Thank you

**Comment by mriscoc** [comment]
Open Cura and go to the Help menu and select Show Configuration Folder, then scripts, put the script file there. Do not use Cura 5.5, that version breaks the compatibility with some third-party scripts.
